pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -387,8 +387,8 @@ func (c *Cluster) enableServiceMonitor(service *v1.Service) error {
 	return nil
 }
 
-// deploy prometheusRule that adds alerting and/or recording rules to the cluster
-func (c *Cluster) deployPrometheusRule(name, namespace string) error {
+// DeployPrometheusRule deploy prometheusRule that adds alerting and/or recording rules to the cluster
+func (c *Cluster) DeployPrometheusRule(name, namespace string) error {
```

Comment:
Currently, there is no Rule file for external mode. This might throw an error. I can send in a patch with this PR for the basic alerts which we can use to test this.